tests: mimxrt1170_evk@B/mimxrt1176/cm7: tests.bluetooth.classic.sdp_s build failure

**Describe the bug**
mimxrt1170_evk@B/mimxrt1176/cm7: bluetooth.classic.sdp.server build failure
git bisect points to the commit that introduced the test. [commit](https://github.com/zephyrproject-rtos/zephyr/commit/9b8297545aa166b5615559d9e617f2a6fadf82cd)

**To Reproduce**
west update
west build -p always -b mimxrt1170_evk@B/mimxrt1176/cm7 .\tests\bluetooth\classic\sdp_s -T bluetooth.classic.sdp.server

**Expected behavior**
build passes.

**Impact**
blocking PR #87423 
shell CI

**Logs and console output**
```
CMake Warning (dev) at C:/WorkFolder/zephyrproject/zephyr/cmake/modules/extensions.cmake:1829 (file):
  Given path:

    C:/WorkFolder/zephyrproject/modules/hal/nxp/zephyr/blobs/iw612/uart_nw61x_se.h

  does not refer to an existing path on disk.
Call Stack (most recent call first):
  C:/WorkFolder/zephyrproject/zephyr/modules/hal_nxp/bt_controller/CMakeLists.txt:22 (zephyr_blobs_verify)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at C:/WorkFolder/zephyrproject/zephyr/cmake/modules/extensions.cmake:1838 (message):
  Blob for path
  "C:\WorkFolder\zephyrproject\modules\hal\nxp\zephyr\blobs\iw612\uart_nw61x_se.h"
  missing.  Update with: west blobs fetch
Call Stack (most recent call first):
  C:/WorkFolder/zephyrproject/zephyr/modules/hal_nxp/bt_controller/CMakeLists.txt:22 (zephyr_blobs_verify)
```

--- a/tests/bluetooth/classic/sdp_c/testcase.yaml
+++ b/tests/bluetooth/classic/sdp_c/testcase.yaml
@@ -2,7 +2,6 @@ tests:
   bluetooth.classic.sdp.client:
     platform_allow:
       - native_sim
-      - mimxrt1170_evk@B/mimxrt1176/cm7
     integration_platforms:
       - native_sim
     tags:
@@ -13,3 +12,13 @@ tests:
       pytest_dut_scope: session
       fixture: usb_hci
     timeout: 180
+  bluetooth.classic.sdp.client.no_blobs:
+    platform_allow:
+      - mimxrt1170_evk@B/mimxrt1176/cm7
+    tags:
+      - bluetooth
+      - sdp
+    extra_args:
+      - CONFIG_BUILD_ONLY_NO_BLOBS=y
+    timeout: 180
+    build_only: true

--- a/tests/bluetooth/classic/sdp_s/testcase.yaml
+++ b/tests/bluetooth/classic/sdp_s/testcase.yaml
@@ -2,7 +2,6 @@ tests:
   bluetooth.classic.sdp.server:
     platform_allow:
       - native_sim
-      - mimxrt1170_evk@B/mimxrt1176/cm7
     integration_platforms:
       - native_sim
     tags:
@@ -13,3 +12,13 @@ tests:
       pytest_dut_scope: session
       fixture: usb_hci
     timeout: 60
+  bluetooth.classic.sdp.server.no_blobs:
+    platform_allow:
+      - mimxrt1170_evk@B/mimxrt1176/cm7
+    tags:
+      - bluetooth
+      - sdp
+    extra_args:
+      - CONFIG_BUILD_ONLY_NO_BLOBS=y
+    timeout: 60
+    build_only: true

